src/item.h

```diff
@@ -1364,7 +1367,11 @@ class item : public JsonSerializer, public JsonDeserializer, public visitable<it
         * Returns label from "item_label" itemvar and quantity
         */
         std::string label( unsigned int quantity = 0 ) const;
+
     private:
+        /** unique identifier for each non-null instance */
+        item_uid uid_;
```

Comment:
I'm quite insistent about this BTW, when one out of a million items needs a uuid, you don't add a uuid to every item, especially now that's a huge 128-bit number.